### DISCUSSION TITLE
Pending mpool messages

### DISCUSSION THREAD

**Discussion by hcm609** [filecoin-project/lotus#7632] [General Q&A] [answered]
I noticed I have a lot of pending messages in mpool.
Here are some stats:

#lotus mpool pending|grep Nonce |wc -l
1932

#lotus mpool pending|wc -l
35496

Is it a matter of wrong Gas fees? If yes, which are the values I should use?

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
add the `--local` flag to the command to see what are actually your messages. ignore the rest - it's others peoples messages

https://docs.filecoin.io/mine/lotus/message-pool/

**Comment by hcm609** [comment]
the --local is empty. 
thank you.
